src/react-native/11-navigation/navigation.md

```diff
@@ -110,7 +110,9 @@ The `Tab.Screen` component takes a `name` prop that is used to identify the scre
 ✏️ Install the `@react-navigation/native` and `@react-navigation/bottom-tabs` packages:
 
 ```bash
-npm install @react-navigation/native @react-navigation/bottom-tabs
+npm install react-navigation/native@6 react-navigation/bottom-tabs@6 react-native-safe-area-context@4
```

Comment:
These are the wrong packages (I think they should be from `@react-navigation`).

Also, I would split up the commands for installing the dev vs. normal dependencies.

Reply:
yes, those need the leading `@`